docs/guides/supported_backends.md

```diff
@@ -0,0 +1,51 @@
+# Supported Backends 
+
+
+GuideLLM requires an OpenAI-compatible server to run evaluations. [vLLM](https://github.com/vllm-project/vllm) is recommended for this purpose; however, GuideLLM is compatible with many backend inference servers such as TGI, llama.cpp, and DeepSparse. 
```

Comment:
interesting position. Ideally, this tool would emerge as an industry standard benchmarking tool. In this case, shouldn't all inferencing services be on the same footing?

Reply:
We are vLLM bias as you know, but yes want to keep the extensibility of supporting multiple backends. Hence the README has a vLLM example, but we created this supported backends.md documentation as well.